tox.ini

```diff
@@ -38,6 +38,7 @@ deps =
     pytest-operator
     aiohttp
     ipdb
+    git+https://github.com/canonical/kubernetes-rapper@main#egg=kubernetes-wrapper
```

Comment:
NBD, but I generally prefer the archive link instead, since it doesn't require git to be involved.  I.e.: `https://github.com/canonical/kubernetes-rapper/archive/refs/heads/main.zip+egg=kubernetes-wrapper`